bakery/README.md

```diff
@@ -6,13 +6,27 @@
 - Run `yarn install` or `yarn`.
 
 ### Generate a pipeline file for a particular environment
-- Run `yarn build:<env>`
+Run `yarn build:<env> [output-path]`
 
-  Available environments: `local`, `staging`, `prod`
+Available environments: `local`, `staging`, `prod`
 
-  Example: `yarn build:staging`
+Example: `yarn build:staging`
 
-  Note: Directory for output path must exist. If no path argument is given, the pipeline will output to `pipeline.yml` in the current working directory.
+Note: Directory for output path must exist if given or if no argument is given, the pipeline will output to stdout.
+
+### Generate a standalone task file suitable for `fly execute`
+Run `yarn build:task <task-name> [yaml-object]`
```

Comment:
This output is being generated with the file when i run `yarn build:task fetch-book > tasks/fetch-book.yml`

![image](https://user-images.githubusercontent.com/8730430/74345536-d9455980-4d73-11ea-93aa-7fbfe5bba43c.png)
This also shows up at the bottom:
![image](https://user-images.githubusercontent.com/8730430/74345896-61c3fa00-4d74-11ea-87b8-f7a76255a356.png)

Reply:
I added a CLI, so that we can call the generator how we want, so we don't get undesirable output from the task runner.